src/Module/Api/Mastodon/Accounts/Followers.php

```diff
@@ -55,32 +56,59 @@ protected function rawContent(array $request = [])
 			'limit'    => 40, // Maximum number of results to return. Defaults to 40.
 		], $request);
 
-		$params = ['order' => ['relation-cid' => true], 'limit' => $request['limit']];
-
-		$condition = ['cid' => $id, 'follows' => true];
-
-		if (!empty($request['max_id'])) {
-			$condition = DBA::mergeConditions($condition, ["`relation-cid` < ?", $request['max_id']]);
-		}
-
-		if (!empty($request['since_id'])) {
-			$condition = DBA::mergeConditions($condition, ["`relation-cid` > ?", $request['since_id']]);
-		}
-
-		if (!empty($request['min_id'])) {
-			$condition = DBA::mergeConditions($condition, ["`relation-cid` > ?", $request['min_id']]);
-
-			$params['order'] = ['cid'];
-		}
-
-		$accounts = [];
-
-		$followers = DBA::select('contact-relation', ['relation-cid'], $condition, $params);
-		while ($follower = DBA::fetch($followers)) {
-			self::setBoundaries($follower['relation-cid']);
-			$accounts[] = DI::mstdnAccount()->createFromContactId($follower['relation-cid'], $uid);
+		if (false && $id == Contact::getPublicIdByUserId($uid)) {
```

Comment:
Please remove debug code, this condition is always false and as a result the first statement of the `if` will never be executed.

Reply:
Ooops ...